src/main/webapp/script.js

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Fetches comments from the server and adds them to the DOM. */
+function loadComments() {
+  fetch('/display-comments').then(response => response.json()).then((comments) => {
+    const commentListElement = document.getElementById('comment-list');
+    comments.forEach((comment) => {
+      commentListElement.appendChild(createCommentElement(comment));
+    })
+  });
+}
+
+/** Redirects the user to the result page, with the given parameter for user-query. */
+function redirectToResults(userQuery) {
+  document.location.href = "search-results-test.html?user-query=" + userQuery;
+  return false;
+}
+
+/** Gets the value of the given parameter from the current URL string. */
+function getURLParamVal(param) {
+  // Get the current page URL.
+  var pageURL = window.location.search.substring(1);
+  // Get the variables in the URL.
+  var urlVariables = pageURL.split('&');
+  for (var i = 0; i < urlVariables.length; i++) {
+    var paramName = urlVariables[i].split('=');
+    if (paramName[0] == param) {
+      return paramName[1];
+    }
+  }
+}
+
+/** Fetches results from the server and adds them to the DOM. */
+function getResults(param) {
+  var userQuery = getURLParamVal(param);
+  fetch('/results?user-query=' + userQuery.toUpperCase()).then(response => response.json()).then((results) => {
+    const resultListElement = document.getElementById('result-list');
+    results.forEach((result) => {
+      console.log("Result found!");
```

Comment:
remove print statement if it's just for debugging

Reply:
;-; ok